tools/rpkg/R/relational.R

```diff
@@ -379,3 +379,17 @@ rel_from_table <- function(con, table_name, schema_name="MAIN") {
     rapi_rel_from_table(con@conn_ref, schema_name, table_name)
 }
 
+#' Convert a duckdb relation from a table-producing function
+#' @param name the table function name
+#' @param positional_parameters the table function positional parameters list
+#' @param named_parameters the table function named parameters list
+#' @return a duckdb relation
+#' @noRd
+#' @examples
+#' con <- DBI::dbConnect(duckdb())
+#' DBI::dbWriteTable(con, "mtcars", mtcars)
+#' rel <- rel_from_table(con, "mtcars")
```

Comment:
Need to tweak example:

```suggestion
#' rel <- rel_from_table_function(con, ...)
```